src/Nethermind/Nethermind.Trie.Test/TrieTests.cs

```diff
@@ -29,7 +29,8 @@ public class TrieTests
         [SetUp]
         public void SetUp()
         {
-            _logManager = new NUnitLogManager(LogLevel.Trace);
+            _logManager = LimboLogs.Instance;
+             // new NUnitLogManager(LogLevel.Trace);
```

Comment:
And another one.

Reply:
No the tests are currently flooded with messages from those and while tests are fine they are not needed. They would only be useful when we have regression.